docs/docs/guides/how-to-parameterize-cassandra.md

```diff
@@ -0,0 +1,396 @@
+Service 'cassandra-node-4' added with service UUID '8ecd6c4b75a64764a87f2ce9d23cd8f0'
+
+> wait recipe=ExecRecipe(command=["/bin/sh", "-c", "nodetool status | grep UN | wc -l | tr -d '\n'"]) field="output" assertion="==" target_value="5" timeout="15m"
+Wait took 33 tries (52.686225608s in total). Assertion passed with following:
+Command returned with exit code '0' and the following output: 5
+
+Starlark code successfully run. Output was:
+{
+	"node_names": [
+		"cassandra-node-0",
+		"cassandra-node-1",
+		"cassandra-node-2",
+		"cassandra-node-3",
+		"cassandra-node-4"
+	]
+}
+INFO[2023-03-28T21:46:55-03:00] ==========================================================
+INFO[2023-03-28T21:46:55-03:00] ||          Created enclave: cassandra-cluster          ||
+INFO[2023-03-28T21:46:55-03:00] ==========================================================
+Name:            cassandra-cluster
+UUID:            c7985e32b076
+Status:          RUNNING
+Creation Time:   Tue, 28 Mar 2023 21:45:47 -03
+
+========================================= Files Artifacts =========================================
+UUID   Name
+
+========================================== User Services ==========================================
+UUID           Name               Ports                                  Status
+9a9213d1d846   cassandra-node-0   client: 9042/tcp -> 127.0.0.1:54740    RUNNING
+                                  cluster: 7000/tcp -> 127.0.0.1:54741
+66c6907c6a8a   cassandra-node-1   client: 9042/tcp -> 127.0.0.1:54746    RUNNING
+                                  cluster: 7000/tcp -> 127.0.0.1:54745
+fe9be7991edd   cassandra-node-2   client: 9042/tcp -> 127.0.0.1:54761    RUNNING
+                                  cluster: 7000/tcp -> 127.0.0.1:54760
+daff154657ce   cassandra-node-3   client: 9042/tcp -> 127.0.0.1:54768    RUNNING
+                                  cluster: 7000/tcp -> 127.0.0.1:54769
+8ecd6c4b75a6   cassandra-node-4   client: 9042/tcp -> 127.0.0.1:54773    RUNNING
+                                  cluster: 7000/tcp -> 127.0.0.1:54774
+```
+
+Congratulations! We’ve just parameterized our Cassandra cluster environment definition and used it to instantiate a 5-node Cassandra cluster. You can run the same command with 2 nodes, or 4 nodes and it will work **Kurtosis environment definitions are completely reproducible and fully parametrizable.**
+
+:::caution
+Depending on how many nodes you wish to spin up, the max heap size of each node may cumulatively consume more memory on your local machine than you have available, causing the Starlark script to time-out. Modifying the `MAX_HEAP_SIZE` property may help, depending on your needs.
```

Comment:
@leeederek  can you specify where the user would modify this property? just so its abundantly clear / user doesn't have to think

Reply:
Yup, done!